### ISSUE TITLE
Control speed of animation

### ISSUE THREAD

**Issue by yairhaimo** [stipsan/react-spring-bottom-sheet#30]
Is there a way to control the speed of the animation?
Specifically using the `snapTo` method.

(Awesome, awesome library!)

**Comment by stipsan** [comment]
Heya, [react-spring](https://www.react-spring.io/docs/hooks/api#configs) support it. If I expose it like this:
```js
ref.current.snapTo(({maxHeight}) => maxHeight, {duration: 250})
```
would that work for you?

**Comment by yairhaimo** [comment]
That would be great. Thanks!

**Comment by simonrelet** [comment]
Hello there!
@stipsan Huge thanks for the library 🙏 

Would it be conceivable to also have an API to set the duration/spring config at the component level?
`ref.current.snapTo()` could "use it" by default and override it if a config is passed as parameter.

What do you think?

**Comment by stipsan** [comment]
@simonrelet yes that is the idea. The reason I've held off on it though is that these options are different in the upcoming version of `react-spring`, `v9`. I could compromise on using an `unstable_` prefix on this prop/option while we're on `v8`. If I add it now understand that these options will be a "use at your own risk" kind of deal.

**Comment by otox-rz** [comment]
Hi @stipsan , I think we will have the good news soon .

[Release Plan v9 of react-spring](https://github.com/pmndrs/react-spring/issues/1287)

**Comment by ejabu** [comment]
Hi @stipsan , we have a good news from `React Spring` side.

[Finally the version 9 is merged](https://github.com/pmndrs/react-spring/pull/1330)

**Comment by stipsan** [comment]
Brilliant 🎉  Btw, @otox-rz and @ejabu, what's up with the 🍔 themed avatars? 😂

**Comment by ejabu** [comment]
> Brilliant 🎉 Btw, @otox-rz and @ejabu, what's up with the 🍔 themed avatars? 😂

lol, I had to create another account to conform my office rules, using Work email for git user. 🤣

**Comment by Agrejus** [comment]
FYI - Here is an answer for changing the speed

https://github.com/stipsan/react-spring-bottom-sheet/issues/151#issuecomment-1368631716
